gradlew

```diff
@@ -11,13 +11,13 @@
 PRG="$0"
 # Need this for relative symlinks.
 while [ -h "$PRG" ] ; do
-    ls=`ls -ld "$PRG"`
-    link=`expr "$ls" : '.*-> \(.*\)$'`
-    if expr "$link" : '/.*' > /dev/null; then
-        PRG="$link"
-    else
-        PRG=`dirname "$PRG"`"/$link"
-    fi
+	ls=`ls -ld "$PRG"`
```

Comment:
I would exclude gradlew from any refactoring as it is not a file we edited

Reply:
The idea is that EditorConfig is used to shield these auto-generated vendor files in this case — i.e. they shouldn't be edited, which means that any changes in styles conventions would be an indicator that the file was tampered with.

Reply:
(I didn't change the `gradlew` file, if this is what you meant)

Reply:
OK, done! I've removed the whole Gradle definition from EditorConfig, and reverted the file to its previous state.

(sorry, I had forgot about the edits, I though it was a manually written file).